lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -216,6 +219,25 @@ export abstract class CacheManager implements ICacheManager {
         credential: ValidCredentialType
     ): string;
 
+    /**
+     * Read the user-supplied cache into memory, if applicable
+     * @param serializableCache 
+     * @param persistencePlugin 
+     */
+    async readUserDefinedCacheIntoMemory(
```

Comment:
CacheManager doesn't deal with calling beforeCache / afterCache hooks. Not sure it's the best place to put this in.